rewriter/src/test/java/com/facebook/coresql/rewriter/TestOrderByRewriter.java

```diff
@@ -16,93 +16,41 @@
-            "SELECT dealer_id, sales OVER (PARTITION BY dealer_id ORDER BY sales);",
-            "INSERT INTO blah SELECT * FROM (SELECT t.date, t.code, t.qty FROM sales AS t ORDER BY t.date LIMIT 100);",
-            "SELECT (true or false) and false;",
+            "CREATE TABLE blah AS SELECT * FROM (SELECT * FROM (SELECT foo FROM T ORDER BY x LIMIT 10) ORDER BY y LIMIT 10) ORDER BY z LIMIT 10;", "SELECT dealer_id, sales OVER (PARTITION BY dealer_id ORDER BY sales);", "INSERT INTO blah SELECT * FROM (SELECT t.date, t.code, t.qty FROM sales AS t ORDER BY t.date LIMIT 100);", "SELECT (true or false) and false;",
             // True Negative
-            "SELECT * FROM T ORDER BY y;",
-            "SELECT * FROM T ORDER BY y LIMIT 10;",
-            "use a.b;",
-            " SELECT 1;",
-            "SELECT a FROM T;",
-            "SELECT a FROM T WHERE p1 > p2;",
-            "SELECT a, b, c FROM T WHERE c1 < c2 and c3 < c4;",
-            "SELECT CASE a WHEN IN ( 1 ) THEN b ELSE c END AS x, b, c FROM T WHERE c1 < c2 and c3 < c4;",
-            "SELECT T.* FROM T JOIN W ON T.x = W.x;",
-            "SELECT NULL;",
-            "SELECT ARRAY[x] FROM T;",
-            "SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;",
-            "CREATE TABLE T AS SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;",
-            "INSERT INTO T SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;",
-            "SELECT ROW_NUMBER() OVER(PARTITION BY x) FROM T;",
-            "SELECT x, SUM(y) OVER (PARTITION BY y ORDER BY 1) AS min\n" +
-                    "FROM (values ('b',10), ('a', 10)) AS T(x, y)\n;",
-            "SELECT\n" +
-                    " CAST(MAP() AS map<bigint,array<boolean>>) AS \"bool_tensor_features\";",
-            "SELECT f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f())))))))))))))))))))))))))))));",
-            "SELECT abs, 2 as abs;",
+            "SELECT * FROM T ORDER BY y;", "SELECT * FROM T ORDER BY y LIMIT 10;", "use a.b;", " SELECT 1;", "SELECT a FROM T;", "SELECT a FROM T WHERE p1 > p2;", "SELECT a, b, c FROM T WHERE c1 < c2 and c3 < c4;", "SELECT CASE a WHEN IN ( 1 ) THEN b ELSE c END AS x, b, c FROM T WHERE c1 < c2 and c3 < c4;", "SELECT T.* FROM T JOIN W ON T.x = W.x;", "SELECT NULL;", "SELECT ARRAY[x] FROM T;", "SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;", "CREATE TABLE T AS SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;", "INSERT INTO T SELECT TRANSFORM(ARRAY[x], x -> x + 2) AS arra FROM T;", "SELECT ROW_NUMBER() OVER(PARTITION BY x) FROM T;", "SELECT x, SUM(y) OVER (PARTITION BY y ORDER BY 1) AS min\n" + "FROM (values ('b',10), ('a', 10)) AS T(x, y)\n;", "SELECT\n" + " CAST(MAP() AS map<bigint,array<boolean>>) AS \"bool_tensor_features\";", "SELECT f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f(f())))))))))))))))))))))))))))));", "SELECT abs, 2 as abs;",
```

Comment:
Same comment here - don't concatenate the test sql strings

Reply:
Solved via Commit https://github.com/prestodb/sql/pull/31/commits/24ddd02e68c8658c7285302de9be6c65dc4d59a3